helm/operational-load-exporter/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: A Helm chart for operational-load-exporter
+home: https://github.com/giantswarm/operational-load-exporter
+name: operational-load-exporter
+appVersion: [[ .AppVersion ]]
+version: [[ .Version ]]
```

Comment:
why not use app bundle? it makes this automatic for you

Reply:
@pipo02mix can you show me an example? i'm learning this as i go

Reply:
[Alfred used this one](https://github.com/giantswarm/alfred-app) but the [template-app](https://github.com/giantswarm/template-app) is the reference